src/main/java/com/sublinks/sublinksapi/api/lemmy/v3/user/controllers/UserModActionsController.java

```diff
@@ -106,6 +108,16 @@ BanPersonResponse ban(@Valid @RequestBody final BanPerson banPersonForm, JwtPers
     if (banPersonForm.ban()) {
       personToBan.setRole(
           roleService.getBannedRole(() -> new RuntimeException("No Banned role found.")));
+      if (banPersonForm.expires() != null) {
+//        if (banPersonForm.expires() < System.currentTimeMillis() / 1000) {
+//          throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "expires_in_past");
+//        }
```

Comment:
Why are we leaving in commented out code?

Reply:
My bad